code/datums/intercept_text.dm

```diff
@@ -135,3 +139,9 @@
 	src.text += "<BR><BR>We have intercepted reports that a Space Wizard Federation menagerie facility in your sector has suffered a containment breach. "
 	src.text += "It is possible that a Vampire has escaped from their cells and is likely to have taken refuge on the station. It is likely weak from its "
 	src.text += "extended containment, but it will become increasingly more powerful if allowed to consume human blood. If caught, it must be terminated."
+
+/datum/intercept_text/proc/build_flock(correct_mob)
+	src.text += {"<BR><BR>An anomalous radio entity has been detected by long range sensors in your sector. This entity is likely a Flockmind. From
+				 previous reports, it's believed that it will attempt to use your station's resources to build a massive superstructure to transmit a
+				 radio signal, which is harmful to the station and crew. Its Flock may be indirectly aggressive but it could become a threat at any
+				 time."}
```

Comment:
Personally I'd rather this was a little more oblique than "it is called a Flockmind and it will do all this stuff", see how the changeling one just calls it a "mutagenic organism" and vaguely references some of its powers.

Reply:
Look okay?